README.md

```diff
@@ -159,17 +159,17 @@ ffplay -i unix:/tmp/myvideo.sock
 
 Recording requires [egress service](https://docs.livekit.io/guides/egress/) to be set up first.
 
-Example request.json files are [located here](https://github.com/livekit/livekit-cli/tree/main/cmd/livekit-cli/examples).
+Example request.json files are [located here](https://github.com/livekit/livekit-cli/tree/main/cmd/lk/examples).
 
 ```shell
 # start room composite (recording of room UI)
-livekit-cli start-room-composite-egress --request request.json
+lk start-room-composite-egress --request request.json
 
 # start track composite (audio + video)
-livekit-cli start-track-composite-egress --request request.json
+lk start-track-composite-egress --request request.json
 
 # start track egress (single audio or video track)
-livekit-cli start-track-egress --request request.json
+lk start-track-egress --request request.json
```

Comment:
`lk egress start track`?

Reply:
The way I refactored it, it is now `lk egress start --type track` (when omitted, type defaults to `room-composite`)

Reply:
Also I'm new to Go, but we could also consider doing a tagged union approach where `request.json` has a `type` field we can read and know implicitly the kind of struct to unmarshal, and what operation you're doing without having to be redundant.